python/graphstorm/gconstruct/utils.py

```diff
@@ -1118,6 +1118,28 @@ def get_hard_edge_negs_feats(hard_edge_neg_ops):
 
     return hard_edge_neg_feats
 
+def get_gnid2pnid_map(ntype, node_mapping, gnid2pnid_mapping):
+    """ Get global nid to partitioned nid mapping.
+
+        Parameters
+        ----------
+        ntype: str
+            Path to the directory storing the partitioned graph.
+        node_mapping: dict
+            Dict of mapping. {ntype: partitioned nid to global nid mapping}
```

Comment:
Add a few more details for the contents? How are the mappings represented (torch Tensor where row ids mean this and row values mean that)